taskcluster/mozillavpn_taskgraph/transforms/build.py

```diff
@@ -66,4 +66,15 @@ def resolve_keys_scope(config, tasks):
                 "level": config.params["level"],
             },
         )
-        yield task
\ No newline at end of file
+        yield task
+
+
+@transforms.add
+def add_pr_number(config, tasks):
+    for task in tasks:
+        if config.params["pull_request_number"]:
+            task["worker"].setdefault("env", {}).update({
+                "PULL_REQUEST_NUMBER": str(config.params["pull_request_number"])
```

Comment:
Nice! - probably the same could be done for the author "todo",  i think. But that's a future (tm) thing :)